frontend/src/App.tsx

```diff
@@ -1,43 +1,17 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
 import './App.css';
-import {NewTodo, Todo} from "./model/Todo";
-import axios from "axios";
 import TodoBoards from "./components/TodoBoards";
 import AddTodo from "./components/AddTodo";
+import useTodos from "./hooks/useTodos";
+import axios from "axios";
 
 function App() {
 
-    const [todos, setTodos] = useState<Todo[]>([])
-
-    useEffect(() => {
-        loadAllTodos()
-    }, [])
-
-    function loadAllTodos() {
-        return axios.get("/api/todo")
-            .then(response => response.data)
-            .then(setTodos)
-            .catch(console.error)
-    }
-
-    function addTodo(todo: NewTodo) {
-        return axios.post("/api/todo", todo)
-            .then(response => response.data)
-            .then(data => setTodos(prevState => [...prevState, data]))
-    }
+    const { todo, addTodo, updateTodo } = useTodos()
 
-    function updateTodo(todo: Todo) {
-        axios.put("/api/todo/" + todo.id, todo)
-            .then(response => response.data)
-            .then(data =>  setTodos(prevState => {
-                return prevState.map(currentTodo => {
-                    if (currentTodo.id === todo.id) {
-                        return data
-                    }
-                    return currentTodo
-                })
-            }))
-    }
+    // function deleteTodo(id: string) {
```

Comment:
Is the code needed?
Please remove code that is not currently used for this pull request.